openpype/pipeline/load/utils.py

```diff
@@ -286,6 +288,54 @@ def get_representation_context(representation):
     return context
 
 
+def get_representation_files(context, filepath):
+    """Return list of files for representation.
+
+    Args:
+        representation (dict): Representation document.
+        filepath (str): Filepath of the representation.
+
+    Returns:
+        list[str]: List of files for representation.
+    """
+    version = context["version"]
+    frame_start = version["data"]["frameStart"]
+    frame_end = version["data"]["frameEnd"]
+    handle_start = version["data"]["handleStart"]
+    handle_end = version["data"]["handleEnd"]
```

Comment:
Note: Not all versions have frame range data.
Also, representation can also store frame range data that supersedes the version data, so that should be taken into account.